### PR TITLE
AP1639 income/outgoing on caseworker csv report

### PR DESCRIPTION
**Pull request by Obsiye**
AP1639 income/outgoing on caseworker csv report

[Link to story](https://dsdmoj.atlassian.net/browse/AP-1639)

Update the bank_transaction_report csv with income and outgoing and the selector e.g. provider or system

## Checklist

Before you ask people to review this PR:

- [x] Tests and rubocop should be passing: `bundle exec rake`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- [x] The PR description should say what you changed and why, with a link to the JIRA story.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: spec/presenters/bank_transaction_presenter_spec.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@
     subject(:headers) { described_class.headers }
 
     it { is_expected.to be_a Array }
-    it { expect(headers.count).to eq 9 }
+    it { expect(headers.count).to eq 10 }
```
Comment (mpw5) [inline spec/presenters/bank_transaction_presenter_spec.rb:15]: Just a heads up that I've added a couple of extra columns to this report in another [PR](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/1744) so, depending on which of us merges to master first, you might need to rebase and change the value here.

**Review by willc-work — APPROVED** [review]

**Review by mpw5 — APPROVED** [review]
Good work 🎉  I've made one comment about potential conflicts with some similar changes I'm making, just something to be aware of. Happy to work with you to resolve them if needed.

File: spec/presenters/bank_transaction_presenter_spec.rb (hunk 1 above)
Comment (Obsiye) [reply spec/presenters/bank_transaction_presenter_spec.rb:15]: thank you, I'll make the updates after you merge

**Review by colinbruce — APPROVED** [review]
LGTM :+1:

**Pull request merged**